src/app/core/user/eua/eua.service.js

```diff
@@ -35,19 +35,12 @@ const search = async (queryParams, query, _search) => {
 	query = query || {};
```

Comment:
Another github action warning, as it looks like we're no longer using async/await.